modules/core/client/app/init.js

```diff
@@ -12,7 +12,23 @@ angular.module(ApplicationConfiguration.applicationModuleName).config(['$locatio
   }
 ]);
 
-angular.module(ApplicationConfiguration.applicationModuleName).run(function ($rootScope, $state, Authentication) {
+angular.module(ApplicationConfiguration.applicationModuleName).run(function ($rootScope, $state, $http, Authentication, CORE_CONST) {
+
+  //set token header
+  var auth_token = localStorage.getItem('auth_token');
```

Comment:
I've personally stored tokens in local storage for all my projects. That's mainly where I've seen it anywhere else as well.